pkg/apis/apps/types.go

```diff
@@ -502,8 +502,7 @@ type DeploymentList struct {
 
 // DaemonSetUpdateStrategy defines a strategy to update a daemon set.
 type DaemonSetUpdateStrategy struct {
-	// Type of daemon set update. Can be "RollingUpdate" or "OnDelete".
-	// Default is OnDelete.
+	// Type of daemon set update. Can be "RollingUpdate" or "OnDelete". Default is RollingUpdate.
```

Comment:
defaults are specific to external versions (e.g. apps/v1, etc). remove the info about the default from this internal type

Reply:
sure, PR updated :)